runtime/README.md

```diff
@@ -0,0 +1,488 @@
+  be read out over JTAG
+
+Table: `GET_LDEV_CERT` output arguments
+
+| **Name**  | **Type**      | **Description**
+| --------  | --------      | ---------------
+| data      | u8[...]       | DER-encoded LDevID Certificate
+
+### ECDSA384\_SIGNATURE\_VERIFY
+
+Command Code: `0x5349_4756` ("SIGV")
+
+Table: `ECDSA384_SIGNATURE_VERIFY` input arguments
+
+| **Name**     | **Type** | **Description**
+| --------     | -------- | ---------------
+| crc          | u32      | CRC over other input arguments, computed by the caller. Little endian.
+| data         | u8[48]   | Signed hash to verify
+| pub\_key\_x  | u8[48]   | X portion of ECDSA verification key
+| pub\_key\_y  | u8[48]   | Y portion of ECDSA verification key
+| signature\_r | u8[48]   | R portion of signature to verify
+| signature\_s | u8[48]   | S portion of signature to verify
+
+
+Table: `ECDSA384_SIGNATURE_VERIFY` output arguments
+
+| **Name** | **Type** | **Description**
+| -------- | -------- | ---------------
+| crc      | u32      | CRC over other output arguments, computed by Caliptra. Little endian.
+| result   | u32      | Result code. Little endian.
+
+### STASH\_MEASUREMENT
+
+Make a measurement into the DPE default context. This command is intendend for
+callers who update infrequently and cannot tolerate a changing DPE API surface.
+
+Internally, this will call the DPE DeriveChild command.
+
+Command Code: `0x4D45_4153` ("MEAS")
+
+Table: `STASH_MEASUREMENT` input arguments
+
+| **Name**     | **Type** | **Description**
+| --------     | -------- | ---------------
+| crc          | u32      | CRC over other input arguments, computed by the caller. Little endian.
+| metadata     | u8[4]    | 4-byte measurement identifier.
+| measurement  | u8[48]   | Data to measure into DPE.
+
+
+Table: `STASH_MEASUREMENT` output arguments
+
+| **Name**    | **Type** | **Description**
+| --------    | -------- | ---------------
+| crc         | u32      | CRC over other output arguments, computed by Caliptra. Little endian.
+| result      | u32      | Result code. Little endian.
+| dpe\_result | u32      | Result code of DPE DeriveChild command. Little endian.
+
+### INVOKE\_DPE\_COMMAND
+
+Command Code: `0x4450_4543` ("DPEC")
+
+Table: `INVOKE_DPE_COMMAND` input arguments
+
+| **Name**     | **Type**      | **Description**
+| --------     | --------      | ---------------
+| crc          | u32           | CRC over other input arguments, computed by the caller. Little endian.
+| data         | u8[...]       | DPE command structure as defined in the DPE iRoT profile
+
+
+Table: `INVOKE_DPE_COMMAND` output arguments
+
+| **Name**    | **Type**      | **Description**
+| --------    | --------      | ---------------
+| crc         | u32           | CRC over other output arguments, computed by Caliptra. Little endian.
+| result      | u32           | Result code. Little endian.
+| data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
+
+## CRC
```

Comment:
Since this is not the definition of a CRC, should we call it checksum instead or use CRC32?

Reply:
Makes sense. Changed to Checksum.